README.md

```diff
@@ -27,3 +64,20 @@ TEST_CASE=tests/broker/kafka/p10-r3-ordered ./bin/run_test.sh
   --payload-file payloads/payload.68KB.txt \
   --delivery-order ordered
 ```
+
+## Notes:
+- Upstream nightly manifests pushed by a bot (https://github.com/openshift-knative/eventing-hyperfoil-benchmark/blob/main/.github/workflows/scheduled-update-nightlies.yaml)
+
+- There is no automation for product and product-nightly manifests yet
```

Comment:
This is not true, both are pointing to the latest already of their respective stream.

Reply:
Ok, got it. That's done by subscriptions. Thanks!